Makefile

```diff
@@ -548,103 +569,125 @@ $(TOOLCHAIN):
 	git config --global core.autocrlf false
 	mkdir -p $(TOOLCHAIN)
 
+$(PATCH_DIR):
```

Comment:
Should not that be a part of either pass1 or pass2 of gcc?
Also, I have other patches that should go into src/patches, so this task won't probably wokr for other patches (some prereqs will be missing). 
Thoughts?